app/modules/ads/templates/roadblock-config.js

```diff
@@ -0,0 +1,12 @@
+export const getConfig = () => (
+  {
+    slotsToEnable: [
+      'incontent_boxad_1',
+      'mobile_in_content',
```

Comment:
What about bottom leaderboard? How is it enabled?
I know I sometimes ask stupid/repeated questions and thank you for your patience!

Reply:
There should be only +1 slot in roadblock (along with TLB). Nef decided to enable incontent one (when AE3 is enabled)